be/src/runtime/memory/thread_mem_tracker_mgr.h

```diff
@@ -161,9 +163,9 @@ inline void ThreadMemTrackerMgr::pop_consumer_tracker() {
     _consumer_tracker_stack.pop_back();
 }
 
-inline void ThreadMemTrackerMgr::consume(int64_t size, bool large_memory_check) {
+inline void ThreadMemTrackerMgr::consume(int64_t size, int skip_large_memory_check) {
```

Comment:
why change the input parameter from bool to int?

Reply:
Because `int` can support nesting, Maybe there might be a better way?